mdx2/data.py

```
@@ -71,11 +71,11 @@ def from_nexus(peaks):
 class HKLTable:
     """Container for data in a table with indices h,k,l"""
 
-    def __init__(self, h, k, l, ndiv=[1, 1, 1], **kwargs):
+    def __init__(self, h, k, l, ndiv=(1, 1, 1), **kwargs):
```

Ruff (0.14.8)

Ambiguous variable name: `l`

(E741)